tests/integration/test_interaction.py

```diff
@@ -311,8 +311,8 @@ def alt_text_keyboard_controls(self, items_map):
             self.assertEqual(item.get_attribute('aria-describedby'), keyboard_help_text)
 
     def alt_text_for_occupied_zones(self, items_map, scroll_down=100, action_key=None):
-        popup = self._get_popup()
-        feedback_popup_content = self._get_popup_content()
+        self._get_popup()
+        self._get_popup_content()
```

Comment:
There's a strange situation where this code is either failing the run_tests.py (for an aria-describedby being None) or pylint (for unused variables) tests, while the only thing I'm changing is this. I'll looking into the aria-describedby being none, but any insight as to why changing whether I'm assigning these objects to variables might affect things?